### DISCUSSION TITLE
Modelling solar load through glass window

### DISCUSSION THREAD

**Discussion by elangsz** [firemodels/fds#13573] [Q&A] [answered]
hello, im new to FDS and i would like to ask a question. how can i model solar load (sun radiation) in FDS as a boundary condition?  in this case i already obtain the solar heat gain from a separate software which took into account the solar radiation intensity, solar radiation angle, glass transmittance and glass area (assumed to have no exterior shading). is it close enough if we just put in the net heat flux in the boundary condition in FDS?

**Comment by drjfloyd** [comment, marked as answer]
You should think about what exactly is your solar model giving you as output and how well the assumptions made in that model will match the conditions in your FDS calculation:

-Are you just getting radiant energy transmission throuhg glass from your calculation? If so how will you handle convection from sun exposed glass?
-Do you need to account for solar insolation on the exterior walls?
-What are you simulating with FDS?  Trying to model performance of an HVAC system where interior temperatures are not going to vary significantly and stay consistent with the assumptions in your solar model? Or do you have a fire where insde temperatures will rise and cause the net heat flux at a window to be outward?
-etc.
